app/src/test/java/com/example/myspacexdemoapp/ui/launches/LaunchesViewModelTest.kt

```diff
@@ -32,17 +33,20 @@ class LaunchesViewModelTest : TestCase() {
 
     @Test
     fun `when launch by id initialized then success is retrieved`() {
-        val mockResponse: Response<GetLaunchesQuery.Data> =
-            mock(Response::class.java) as Response<GetLaunchesQuery.Data>
+        val mockResponse: ApolloResponse<GetLaunchesQuery.Data> =
```

Comment:
Apollo provides new way to mock data response.

Reply:
`val data = GetLaunchesQuery.Data {
            launches = listOf(
                launch {
                    id = "1111"
                    details = "My details"
                    mission_name = "My mission name"
                }
            )
        }`

Reply:
https://www.apollographql.com/docs/kotlin/advanced/test-builders/